managed/src/test/java/com/yugabyte/yw/common/ModelFactory.java

```diff
@@ -145,20 +145,20 @@ public static Universe createUniverse(String universeName, UUID universeUUID, lo
   }
 
   public static CustomerConfig createS3StorageConfig(Customer customer) {
-    JsonNode formData = Json.parse("{\"name\": \"S3\", \"type\": \"STORAGE\", \"data\": " +
+    JsonNode formData = Json.parse("{\"configName\": \"TEST\", \"name\": \"S3\", \"type\": \"STORAGE\", \"data\": " +
```

Comment:
We have strict line length limitation to 100 characters. Please update here and long lines below.